gcc/rust/hir/rust-ast-lower-base.cc

```diff
@@ -530,16 +530,24 @@ ASTLoweringBase::visit (AST::SelfParam &param)
 {}
 
 HIR::Lifetime
-ASTLoweringBase::lower_lifetime (AST::Lifetime &lifetime)
+ASTLoweringBase::lower_lifetime (AST::Lifetime &lifetime,
+				 bool default_to_static_lifetime)
 {
+  auto lifetime_type = lifetime.get_lifetime_type ();
+  if (lifetime_type == AST::Lifetime::WILDCARD && default_to_static_lifetime)
+    {
+      // If compiling in a static context.
+      lifetime_type = AST::Lifetime::STATIC;
+    }
```

Comment:
```suggestion
  // If compiling in a static context.
  if (lifetime_type == AST::Lifetime::WILDCARD && default_to_static_lifetime)
      lifetime_type = AST::Lifetime::STATIC;
```

GNU style nit, sorry